src/screens/ContactScreen/index.js

```diff
@@ -192,59 +192,57 @@ const ContactScreen = ({navigation}) => {
-        />
+      <StatusBar translucent={false} />
+      <Header
+        title={StringConstant.chatTabHeaderCreateChatButtonText}
+        containerStyle={styles.containerStyle}
+        subTitle={'Next'}
+        onPressSub={() => handleCreateChannel()}
+        onPress={() => navigation.goBack()}
+        disabledNextBtn={selectedUsers.length <= 0}
+      />
 
-        <Search
-          text={text}
-          style={styles.containerStyle}
-          onChangeText={onSearchTextChange}
-          onClearText={() => onSearchTextChange('')}
-          isLoading={isLoadingSearchResult}
-          // onPress={handleSearch}
-        />
+      <Search
+        text={text}
+        style={styles.containerStyle}
+        onChangeText={onSearchTextChange}
+        onClearText={() => onSearchTextChange('')}
+        isLoading={isLoadingSearchResult}
+        // onPress={handleSearch}
+      />
 
-        <View>
-          {selectedUsers && (
-            <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
-          )}
-        </View>
-
-        {isRecyclerViewShown && !isSearchMode && (
-          <RecyclerListView
-            style={styles.recyclerview}
-            layoutProvider={layoutProvider}
-            dataProvider={dataProvider}
-            extendedState={{
-              followed
-            }}
-            rowRenderer={rowRenderer}
-            scrollViewProps={{
-              refreshControl: <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
-            }}
-          />
-        )}
-
-        {isSearchMode && (
-          <SearchRecyclerView
-            text={debouncedText}
-            followed={followed}
-            selectedUsers={selectedUsers}
-            usernames={usernames}
-            setLoading={setIsLoadingSearchResult}
-            onHandleSelected={(value) => handleSelected(value)}
-          />
+      <View>
+        {selectedUsers && (
+          <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
         )}
-        <Loading visible={loading || loadingCreateChat} />
       </View>
+
+      {isRecyclerViewShown && !isSearchMode && (
+        <RecyclerListView
+          style={styles.recyclerview}
+          layoutProvider={layoutProvider}
+          dataProvider={dataProvider}
+          extendedState={{
+            followed
+          }}
+          rowRenderer={rowRenderer}
+          scrollViewProps={{
+            refreshControl: <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
+          }}
+        />
+      )}
+
+      {isSearchMode && (
+        <SearchRecyclerView
+          text={debouncedText}
+          followed={followed}
+          selectedUsers={selectedUsers}
+          usernames={usernames}
+          setLoading={setIsLoadingSearchResult}
+          onHandleSelected={(value) => handleSelected(value)}
+        />
+      )}
+      <Loading visible={loading || loadingCreateChat} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are mostly about replacing the root `<View>` component with a `<SafeAreaView>`. This is a good practice as it ensures that the content of the screen will not overlap with system-level interface elements on devices with notches or rounded corners. However, I noticed that you have removed the `{flex: 1}` style from the root container. This could potentially cause layout issues depending on the contents of the screen. If you want your content to take up the full available space, you should add this back in.

```diff
- <SafeAreaView style={styles.container}>
+ <SafeAreaView style={[styles.container, {flex: 1}]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->